CMakeLists.txt

```diff
@@ -142,7 +144,7 @@ endif()
 if (X265_FOUND AND NOT WITH_X265_PLUGIN)
     list(APPEND REQUIRES_PRIVATE "x265")
 endif()
-if ((AOM_DECODER_FOUND AND NOT WITH_AOM_DECODER_PLUGIN) OR (AOM_ENCODER_FOUND AND NOT WITH_AOM_ENCODER_PLUGIN))
+if (AOM_FOUND AND NOT (WITH_AOM_DECODER_PLUGIN AND WITH_AOM_ENCODER_PLUGIN))
```

Comment:
Actually, I now realize there was a special case for `AOM_DECODER_FOUND` and `AOM_ENCODER_FOUND` variables and `if (${variableName}_FOUND)` above... However, that doesn't work for system supplied JPEG and OpenJPEG CMake find modules (hence the change to `${packageName}_FOUND`), so we now have a bit of a conflict to be solved somehow...